src/ctapipe/fitting.py

```diff
@@ -102,9 +119,14 @@ def _lts_single_sample(X, y, sample_size, max_iterations, eps=1e-12):
     return beta, error
 
 
-@njit()
 def lts_linear_regression(
-    x, y, samples=20, relative_sample_size=0.85, max_iterations=20, eps=1e-12
+    x,
+    y,
+    samples=20,
+    relative_sample_size=0.85,
+    max_iterations=20,
+    eps=1e-12,
+    rng=FIT_RNG,
```

Comment:
this means that where this is used (e.g. in impact.py or image/timing.py) the RNG will always be initialized with a fixed `seed=0`, not `seed=None`.    It means the random sequence used will always be the same if re-run.   Is that an issue in real use?  It's nice for tests, but maybe not for reality?

Reply:
The seed is the same for each start of ctapipe. So we also process a real data file re-producibly if the same config is used. I think this is the good default and doing otherwise could lead to very surprising behavior and hard to debug issues.

I don't think this is an issue, we are not creating simulations. We are sampling which points to use for a linear regression fit for the image parameters.

It's also not a behavior change (it just uses it's own RNG (already better!) than mutating a global numba-rng instance.